### DISCUSSION TITLE
TAB handling?

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [helix-editor/helix#1291] [Q&A]
Is there any configuration option how Helix handles tabs? Like replace with spaces, tab size, etc.?

**Comment by kirawi** [comment]
I think you can edit in `languages.toml`, or `:indent-style`. It will try to auto-detect indenting based on the open document, with a `languages.toml` fallback.

Note: I'm not sure if `:indent-style` will replace the current indentation for that document.

**Comment by marcosps** [comment]
Is there a global option to force it to use tab if there is definition of a language on `languages.toml`?
